src/webview/mod.rs

```diff
@@ -89,6 +89,10 @@ pub struct WebViewAttributes {
   /// Initialize javascript code when loading new pages. When webview load a new page, this
   /// initialization code will be executed. It is guaranteed that code is executed before
   /// `window.onload`.
+  ///
+  /// ## Platform-specific
+  ///
+  /// - **Android:** The initialization scripts are only executed on custom protocol URLs.
```

Comment:
Okay, I've searched react native and ionic. They both seem to set script in html too.
I think we should document the behavior here too. So users can expect how is it done.